scipy/stats/tests/common_tests.py

```diff
@@ -308,15 +308,17 @@ def check_pickling(distfn, args):
 
 def check_freezing(distfn, args):
     # regression test for gh-11089: freezing a distribution fails
-    # if loc and/or scale are specified
+    # if loc and/or scale are specified. Modified to resolve gh-13225;
+    # loc and scale _should_ affect `a` and `b`
```

Comment:
I can't tell from the diff in this page alone; is this effecting a change if the distribution is ***not*** frozen? 
NOTE: I'm assuming this should only impact the frozen distributions. 

Would it make sense to confirm with a test (either in the test in this PR or a new one, either is fine to me) the uniform case as given by the associated issue (e.g. check it impacts frozen and doesn't imact non-frozen)?

Also, the comment is a bit confusing on clarifying this, suggest to rephrase comment to make it explicit if this works on (a) only frozen, (b) frozen and non-frozen.